content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_addon_all.md

```diff
@@ -76,11 +76,11 @@ kubeadm alpha phase addon all [flags]
       <td colspan="2">--kubeconfig string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/admin.conf"</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">The KubeConfig file to use when talking to the cluster</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">The KubeConfig file to use when talking to the cluster. If the flag is not set, a set of standard locations are searched for an existing KubeConfig file.</td>
     </tr>
 
     <tr>
-      <td colspan="2">--kubernetes-version string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "stable-1.10"</td>
+      <td colspan="2">--kubernetes-version string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "stable-1.11"</td>
```

Comment:
yeah, so that's basically the problem i was talking about on slack.
the version in the whole docs is wrong because this `k/k` PR is not merged yet:
https://github.com/kubernetes/kubernetes/pull/68789

we have two options:
A) merge this now and send a PR to fix these versions later.
B) apply the fix in this PR and hope we don't have to re-generate the docs, because that would remove the fix.

in the next k8s release this will hopefully be handled differently, i hope.

/cc @timothysc